### PR TITLE
Reuse sample loaded successfully step in features

### PR DESCRIPTION
**Pull request by liamtoozer**
# Motivation and Context

A [PR comment](https://github.com/ONSdigital/census-rm-acceptance-tests/pull/159#discussion_r368419841) was raised to point out that a consolidated `sample load successful` step can be reused to check the necessary `CASE_CREATED` and `UAC_UPDATED` events are emitted correctly. 

This same step can be reused in a few different features where we don't need to know about the questionnaire type in the emitted messages. This should help condense some of our features to be less wordy as well.

# What has changed

Updated various features to reuse the 'sample loaded successfully' step

# How to test?

Ensure the tests pass as expected

# Links

https://trello.com/c/lRUqvoW8/516-acceptance-tests-reuse-sample-load-success-step

### CODE REVIEW

**Comment by liamtoozer** [comment]
[Acceptance Tests: Reuse sample load success step (1?)](https://trello.com/c/lRUqvoW8/516-acceptance-tests-reuse-sample-load-success-step-1)

**Review by NickGrantX — APPROVED** [review]
ATs pass. Nice refactoring - very neat! 👍

File: acceptance_tests/features/unaddressed.feature [hunk 1]
```diff
@@ -19,9 +19,9 @@ Feature: Generating UAC/QID pairs for unaddressed letters & questionnaires
     And a Questionnaire Linked event is logged
 
   Scenario: Receipt of unlinked unaddressed
-    When an unaddressed message of questionnaire type 01 is sent
-    Then a UACUpdated message not linked to a case is emitted to RH and Action Scheduler
-    And a receipt for the unlinked UAC-QID pair is received
+    Given an unaddressed message of questionnaire type 01 is sent
+    And a UACUpdated message not linked to a case is emitted to RH and Action Scheduler
+    When a receipt for the unlinked UAC-QID pair is received
```
Comment (AdamHawtin) [inline acceptance_tests/features/unaddressed.feature]: I think `When, Then, And, And` actually made more sense for this scenario, it doesn't need a `Given` set up, the action is `an unaddressed message of questionnaire type 01 is sent` and the following 3 steps are all expected results.

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

**Review by AdamHawtin — APPROVED** [review]
👍

**Pull request merged**